src/uct/tcp/tcp_sockcm.c

```diff
@@ -22,9 +23,24 @@ ucs_config_field_t uct_tcp_sockcm_config_table[] = {
    "TCP CM private data length",
    ucs_offsetof(uct_tcp_sockcm_config_t, priv_data_len), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"SEND_BUF", "auto",
+   "TCP CM socket send buffer size",
+   ucs_offsetof(uct_tcp_sockcm_config_t, sockopt_sndbuf), UCS_CONFIG_TYPE_MEMUNITS},
+
+  {"RECV_BUF", "auto",
+   "TCP CM socket receive buffer size",
+   ucs_offsetof(uct_tcp_sockcm_config_t, sockopt_rcvbuf), UCS_CONFIG_TYPE_MEMUNITS},
+
```

Comment:
you mean tcp_iface and tcp_sockcm should have the same values? why?

Reply:
it should have at least same config table entries with same description

Reply:
@yosefe it seems that in the current state tcp_sockcm and tcp_iface cannot have the same name for an env param. i.e. this doesnt work:
```
$./install/bin/ucx_info -f | grep SNDBUF

UCX_TCP_SNDBUF=auto        //tcp_iface
UCX_TCP_CM_SNDBUF=auto     //tcp_sockcm 
```

for the tcp component the UCT_COMPONENT_FLAG_CM flag is set, so m_cm_config has a valid value and isn't NULL - 
https://github.com/openucx/ucx/blob/master/test/gtest/uct/uct_test.cc#L178 
this means that when trying to modify the config for tcp_iface (here for example https://github.com/openucx/ucx/blob/master/test/gtest/uct/test_pending.cc#L25) it goes into here - https://github.com/openucx/ucx/blob/master/test/gtest/uct/uct_test.cc#L501 and modifies the wrong param.

we could change this behavior of course but imho we shouln't have two env params with the same name in the same component. it can easily lead to confusion.
WDYT?

Thanks @dmitrygx for helping out!